tests/components/advantage_air/test_init.py

```diff
@@ -0,0 +1,33 @@
+"""Test the Advantage Air Initialization."""
+
+from homeassistant.config_entries import ENTRY_STATE_LOADED, ENTRY_STATE_SETUP_RETRY
+
+from tests.components.advantage_air import (
+    TEST_SYSTEM_DATA,
+    TEST_SYSTEM_URL,
+    add_mock_config,
+)
+
+
+async def test_async_setup_entry(hass, aioclient_mock):
+    """Test a successful setup entry."""
+
+    aioclient_mock.get(
+        TEST_SYSTEM_URL,
+        text=TEST_SYSTEM_DATA,
+    )
+
+    entry = await add_mock_config(hass)
+    assert entry.state == ENTRY_STATE_LOADED
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6319846)